server/src/main/java/org/eclipse/openvsx/util/UrlUtil.java

```diff
@@ -134,4 +136,20 @@ private static String getBaseUrl(HttpServletRequest request) {
         return url.toString();
     }
 
+    /**
+     * Extract the rest ** of a wildcard path /<segments>/**
+     * @param request incoming request
+     * @return rest of the path
+     */
+    public static String extractWildcardPath(HttpServletRequest request){
+        String path = (String) request.getAttribute(HandlerMapping.PATH_WITHIN_HANDLER_MAPPING_ATTRIBUTE);
+        String bestMatchPattern = (String ) request.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
+        if (path == null || bestMatchPattern == null) {
+            return "";
+        } else {
+            String restOfPath = new AntPathMatcher().extractPathWithinPattern(bestMatchPattern, path);
+            return restOfPath;
+        }
+    }
```

Comment:
This tool is needed because Spring [does not support](https://github.com/spring-projects/spring-framework/issues/18604) path variables containing multiple path segments.